docs/pages/machine-id/getting-started.mdx

```diff
@@ -0,0 +1,120 @@
+------- --------------------------------------------- ----------- -----------------------------------------
+access  {{internal.logins}}                           <all nodes> event:list,read,session:read,list                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
+auditor no-login-6566121f-b602-47f1-a118-c9c618ee5aec             session:list,read,event:list,read                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
+editor                                                            user:list,create,read,update,delete,...
+```
+
+The above output is from a fresh install of Teleport with the default roles, your cluster may have additional roles. In our example, let’s assume you want to give the bot `access` role to allow it to connect to SSH servers within your cluster.
+
+```bash
+$ tctl bots add robot --roles=access
+```
+
+## Step 3/4. Start Machine ID
+
+Now start Machine ID using the `tbot` binary.
+
+```
+$ tbot start \
+   --data-dir=/var/lib/teleport
+   --destination-dir=./machine-id \
+   --token=00000000000000000000000000000000 \
+   --ca-pin=sha256:1111111111111111111111111111111111111111111111111111111111111111 \
+   --auth-server=auth.example.com:3025
+```
+
+Replace the following fields with values from your own cluster.
+
+* `token`: Replace this with the token that was output by the `tctl bots add` command
+* `ca-pin` Replace this with the CA Pin that was output by the `tctl bots add` command
+* `auth-server`: Replace this with the address of your Teleport Auth server.
+<Details scope={["cloud"]} scopeOnly={true}>
```

Comment:
So we don't have repetition with the list above, maybe replace the whole list with a `Tabs` component with `TabItem`s scoped to self-hosted/Cloud.